DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: frictionless
 Title: Read and Write Frictionless Data Packages
-Version: 0.5.0.9003
+Version: 0.5.0.9004
```

Comment:
Just a detail: I think it should be 0.5.1 based on https://r-pkgs.org/release.html#release-version